config.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Theme configuration file.
+ *
+ * @package capsule
+ */
+
+define('WP_CAPSULE_AUTHOR', 'Crowd Favorite');
+define('WP_CAPSULE_THEME_NAME', 'Capsule');
+define('WP_CAPSULE_THEME_SLUG', 'wp-capsule');
+define('WP_CAPSULE_THEME_VERSION', '1.3.0');
+define('WP_CAPSULE_DIR', get_template_directory());
+define('WP_CAPSULE_URI', get_template_directory_uri());
+define('WP_CAPSULE_UPDATER_DOWNLOAD_ID', '');
```

Comment:
what are the extra constants, `WP_CAPSULE_UPDATER_DOWNLOAD_ID` and  `WP_CAPSULE_UPDATER_RENEWAL_URL` required for?